go/runtime/materialize.go

```diff
@@ -185,13 +197,19 @@ func (m *Materialize) RestoreCheckpoint(shard consumer.Shard) (cp pc.Checkpoint,
 	return cp, nil
 }
 
+func (m *Materialize) logErr(err error, message string) {
+	m.shuffleTaskTerm.logPublisher.Log(log.ErrorLevel, log.Fields{
+		"error": err.Error(),
```

Comment:
Good call. I think I have a _slight_ preference toward spelling out `error` for the user-facing logs, but then it seems like it's going to get weird if we have two different spellings in our logs (since user-facing logs are also forwarded to logrus). Overall, I'd be inclined to keep the spelled out `error` and to change occurrences of `err` to `error` in Flow. How's that sound to you?

Reply:
I'm cool with it.